services/resource-usage-tracker/tests/unit/with_dbs/test_api_resource_tracker_service_runs__list_billable.py

```diff
@@ -76,14 +104,97 @@ async def test_list_service_runs_which_was_billed(
     async_client: httpx.AsyncClient,
 ):
     url = URL("/v1/services/-/usages")
-
     response = await async_client.get(
         f'{url.with_query({"product_name": "osparc", "user_id": _USER_ID})}'
     )
     assert response.status_code == status.HTTP_200_OK
     data = response.json()
-    assert len(data["items"]) == 1
-    assert data["total"] == 1
+    assert len(data["items"]) == 2
+    assert data["total"] == 2
 
     assert data["items"][0]["credit_cost"] < 0
     assert data["items"][0]["transaction_status"] in list(CreditTransactionStatus)
+
+
+@pytest.mark.rpc_test()
```

Comment:
what does this marking do? I can't find anything related to it in the code

Reply:
It is just a helper mark for me, as of now RUT supports both RPC and HTTP listing endpoint I have redid one-to-one the HTTP tests from above with RPC endpoint.  At some point I might remove the HTTP listing and I will leave only RPC so I have marked them for my convenient.